packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -250,6 +250,7 @@ class DebuggerController extends DisposableController
   ValueListenable<List<ConsoleLine>> get stdio => _stdio;
 
   IsolateRef isolateRef;
+  bool get isSystemIsolate => isolateRef.isSystemIsolate;
```

Comment:
should this be isolateRef?.isSystemIsolate in case isolateRef is null?

Reply:
Doesn't hurt to be safe. When are we migrating to null safety again...? :-)